hugr-llvm/src/extension/int.rs

```diff
@@ -281,6 +281,22 @@ fn emit_int_op<'c, H: HugrView<Node = Node>>(
                 .as_basic_value_enum()])
         }),
         IntOpDef::ipow => emit_ipow(context, args),
+        // Type args are width of input, width of output
+        IntOpDef::iwiden_u => emit_custom_unary_op(context, args, |ctx, arg, outs| {
+            let [out] = TryInto::<[_; 1]>::try_into(outs)?;
+            Ok(vec![ctx
+                .builder()
+                .build_int_cast_sign_flag(arg.into_int_value(), out.into_int_type(), false, "")?
+                .as_basic_value_enum()])
+        }),
+        IntOpDef::iwiden_s => emit_custom_unary_op(context, args, |ctx, arg, outs| {
+            let [out] = TryInto::<[_; 1]>::try_into(outs)?;
```

Comment:
Don't really need the typing boilerplate.
```suggestion
            let [out] = outs.try_into()?;
            Ok(vec![ctx
                .builder()
                .build_int_cast_sign_flag(arg.into_int_value(), out.into_int_type(), false, "")?
                .as_basic_value_enum()])
        }),
        IntOpDef::iwiden_s => emit_custom_unary_op(context, args, |ctx, arg, outs| {
            let [out] = outs.try_into()?;
```